### PR TITLE
Automated Build Process (Travis CI)

### PR DESCRIPTION
**Pull request by carlosthe19916**
Added
- Travis
- Codecov

### CODE REVIEW

**Comment by carlosthe19916** [comment]
added /configuration/settings.xml file

**Comment by carlosthe19916** [comment]
In order to make the TravisCI builds pass successfully, I recommend merge https://github.com/project-xavier/xavier-analytics/pull/3 before

File: pom.xml [hunk 1]
```diff
@@ -132,4 +132,55 @@
...
+
+    <profiles>
+        <profile>
+            <id>coverage</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.jacoco</groupId>
+                        <artifactId>jacoco-maven-plugin</artifactId>
+                        <version>0.8.4</version>
```
Comment (mrizzi) [inline pom.xml:144]:
```suggestion
                        <version>0.8.4</version>
                        <configuration>
                            <excludes>
                                <exclude>**org.drools.compiler*</exclude>
                            </excludes>
                        </configuration>
```

**Review by mrizzi — COMMENTED** [review]
suggested one change to avoid the `org.jacoco.agent.rt.internal_035b120.asm.MethodTooLargeException: Method too large: org/drools/compiler/lang/DRL6Lexer.mID ()V` exception during the build

**Review by mrizzi — APPROVED** [review]
thanks @carlosthe19916, it works great :+1:

**Pull request merged**